### ISSUE TITLE
Using Pdf.js - simpleviewer : Not able to update the pdf files according to the increment..

### ISSUE THREAD

**Issue by 3gwebtrain** [mozilla/pdf.js#5869]
I would like to show the page according to the `next` and `previous` arrows. for that, I am calling a function to update the pdf files. It works only once the next time nothing updates in the pdf view. my URL get updated each time the user clicks
example : "pdfs/aspire1.pdf", "pdfs/aspire2.pdf", "pdfs/aspire3.pdf" - like so.
here is my function :
```javascript
loadPage : function (e, pNo) { //pNo updates like 1,2,3
			var that = this;
			var scale = this.scale;
			this.pNo = pNo;
			this.newPage = pNo+'page';

			this.pdfUrl =` "pdfs/aspire"+pNo+".pdf";` //updating the url works fine

			console.log(this.pdfUrl); //updates properly.

			var PAGE_TO_VIEW = 1;
			var SCALE = 0.9;

			PDFJS.getDocument(this.pdfUrl).then(function (pdfDocument) {
console.log(this.pdfUrl) //consoles correctly
			  return pdfDocument.getPage(PAGE_TO_VIEW).then(function (pdfPage) {
			    var pdfPageView = new PDFJS.PDFPageView({
			      container: that.pdfContainer.get(0),
			      id: PAGE_TO_VIEW,
			      scale: SCALE,
			      defaultViewport: pdfPage.getViewport(SCALE),
			      textLayerFactory: new PDFJS.DefaultTextLayerFactory()
			    });
				pdfPageView.setPdfPage(pdfPage);
				return pdfPageView.draw();
			  });
			});
		},
```

But nothing updates in the page. what is wrong here.. any one show me the correct way please?

I tried both this methods, but no luck:
`
	PDFViewerApplication.open('pdfs/aspire"+pNo+".pdf');
			PDFView.open('pdfs/aspire"+pNo+".pdf');
`

**Comment by Snuffleupagus** [comment]
Please make sure that you read: https://github.com/mozilla/pdf.js/blob/master/CONTRIBUTING.md; from which I'm quoting the relevant part for this issue:
> If you are developing a custom solution, first check the examples at https://github.com/mozilla/pdf.js#learning and search existing issues. If this does not help, please prepare a short well-documented example that demonstrates the problem and make it accessible online on your website, JS Bin, GitHub, etc. before opening a new issue or contacting us on the IRC channel -- keep in mind that just code snippets won't help us troubleshoot the problem.

**Comment by timvandermeij** [comment]
Closing as incomplete for now. Provide the information requested above to reopen.
